app/assets/javascripts/backbone/views/project_step_view.coffee

```diff
@@ -61,3 +61,4 @@ class MS.Views.ProjectStepView extends Backbone.View
     @$el.replaceWith(data)
     MS.loadingIndicator.hide()
     MS.timelineView.addBlankStep() unless @persisted || @duplicate
+    MS.CalendarView.rerenderEvents()
```

Comment:
Good question. The classes are uppercase. But the timelineView above is actually an instance of a class, hence the lowercase. We should perhaps discuss this.